test/test-helpers/create-entities.js

```diff
@@ -121,7 +121,7 @@ export function createEditor(editorId) {
 		editorAttribs.id = editorId || random.number();
 		editorAttribs.genderId = gender.id;
 		editorAttribs.typeId = editorType.id;
-		editorAttribs.name = internet.userName();
+		editorAttribs.name = otherEditorAttribs?.name || internet.userName();
```

Comment:
Considering I'm already making you change something here (using metabrainzUserId instead of name) let's make it easier to be reusable:

After defining sane defaults, we can merge editorAttribs and otherEditorAttribs and pass the result to new Editor(…
Maybe the simple `new Editor({...editorAttribs,…otherEditorAttribs})` ?